_data/meltano/extractors/tap-sybase/s7clarke10.yml

```diff
@@ -80,6 +84,7 @@ settings:
   kind: string
   label: Tds Version
   name: tds_version
+  value: none
```

Comment:
```suggestion
```
@s7clarke10 it looks like the tap accepts this value or None. You should be able to just exclude the `value` attribute for this. If someone sets it then it will populate in the config.json but otherwise its excluded. `value` is for non-null defaults.

Reply:
Thank you for confirming this.